### PR TITLE
[actiono] Update workflows

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,3 +21,8 @@ jobs:
     uses: smallstep/workflows/.github/workflows/goCI.yml@main
     with:
       os-dependencies: "libpcsclite-dev"
```
Comment (maraino) [inline .github/workflows/ci.yml:23]: We don't do CGO-enabled builds for step-ca, so this dependency is unnecessary.

**Review by maraino — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (dopey) [reply .github/workflows/ci.yml:23]: I tried taking it out and the CI workflows failed in multiple places where the package was expected.

Comment (maraino) [reply .github/workflows/ci.yml:23]: This is because CGO is enabled here, but installing this dependency is okay.

**Review by maraino — APPROVED** [review]
lgtm

**Pull request merged**